### ISSUE TITLE
Port tab under tools is greyed out, but the esp8266 is plugged into the computer

### ISSUE THREAD

**Issue by Zach5678** [spacehuhn/esp8266_deauther#1526]
The port tab is greyed out as shown in the picture and I can not click it and select a port.
![image](https://user-images.githubusercontent.com/99704218/153987386-08a54a04-acdd-468b-8be8-52f6a64ff1bf.png)

I have the esp8266 plugged into the computer, and when I plugged it in, the esp8266 blinked, but did not do the windows noise thing when it gets plugged in. 

I have done everything else up to thing point correct I think, but it can not upload it because of me not being able to select the port Any help would be greatly appreciated since I have been trying to figure this out for a few hours lol.

**Comment by spacehuhn** [comment]
If you don't see a new port appear when you plug in your board, or the menu is grayed out completely, then no new serial port was detected.  
This can be caused by a missing driver or a defective USB cable. 

Learn more: https://blog.spacehuhn.com/espcomm/

**Comment by Zach5678** [comment]
Ok, I looked at the link you sent and I downloaded the driver for my board for the problem, but what do I do with the downloaded file after that? Thanks for helping
Edit: I downloaded it and went into the file and installed it, but nothing changed with the port being greyed out, I think I'm doing something wrong.

**Comment by Zach5678** [comment]
Ok I know I keep doing comments but I think I found the problem the thing I was using to connect the board to the computer wasn't an adapter but just a regular charger for like a Samsung or something. I'm going to get the actual adapter and hopefully the problem will be solved but thanks for your help so far.

**Comment by Zach5678** [comment]
Ok, I finnaly got it to work. Just in case other people had this problem i want to say what i did to fix it. I first went to the link https://blog.spacehuhn.com/espcomm/ and downloaded the driver. I extracted the file and went into it and went to the setup tab downloaded that. I then went to device manager, and pluged in the board. the usb thing popped up and i clicked update when i clicked the usb. then i changed the driver to what i just downloaded and it worked. just in case someone wanted to know what i did to fix it. Thanks
